resolvers/Query.js

```diff
@@ -20,4 +18,10 @@ const Query = {
     },
 }
 
-export default Query;
\ No newline at end of file
+let lang = ""
+
+function setLang(language) {
+    lang = language
+}
+
+export {Query, setLang};
```

Comment:
You use the variable lang as a global variable. When the query is in, the queryAllItem function will get the global variable as argument, right?

Reply:
You mean delete method **setLang(...)** and work directly with **var lang**, everywhere right? I tried at first, but when graphql was trying to change value of **lang** in **index.js** it gave me ```error assignment to constant variable```

Reply:
sorry misunderstood your question xd, yes i pass this global variable to queryAllItem, same happens in the rest api part